webroot/src/components/Users/UserList/UserList.ts

```diff
@@ -80,7 +80,7 @@ class UserList extends mixins(MixinForm) {
             // { value: 'permissions', name: this.$t('account.permissions') },
```

Comment:
Are we commenting these out because they may be supported in the future?

Reply:
Yeah, these have changed somewhat frequently in the early days, and will continue to as API functionality changes. Having to often look them up from history would be a chore. I don't feel strongly though, it you wanted them removed.

Reply:
My personal opinions are more lenient towards commented out code in these circumstances than the traditional best practice take so I have no issue leaving them